html2wp/forms.php

```diff
@@ -0,0 +1,408 @@
+<?php
+/**
+ * Setup the form on activation based on settings
+ *
+ * @package html2wp/simple-wp-starter-theme
+ */
+
+define( 'THEME_DOMAIN', 'html2wp_theme');
+define( 'THEME_DIR', get_template_directory() );
+define( 'HTML2WP_FORM_CREATED', 'html2wp_gf_form_created_once_555' );
+define( 'GRAVITY_VERSION', '1.9.17.5' );
+define( 'GRAVITY_PENDING_INSTALLATION', 'gform_pending_installation' );
+define( 'GRAVITY_RG_VERSION_KEY', 'rg_form_version' );
+
+/**
+ * Returns the current version of gravity forms from wp-premium
+ *
+ * @return gf version
+ */
+function html2wp_get_gf_version() {
+    $changelog = trim(file_get_contents("https://raw.githubusercontent.com/wp-premium/gravityforms/master/change_log.txt"));
+    $verinfo = explode("Version ", $changelog);
+    $verinfo = explode("-", $verinfo[1]);
+    return trim($verinfo[0]);
+}
+
+/**
+ * Creates a gravity for programatically from the form json
+ *
+ * @return formid
+ */
+function html2wp_setup_gravity_contact_form() {
+    /**
+     * Get the form data from form config json
+     * in order to create a new GV Form
+     */
+
+    global $html2wp_settings;
+
+    if ( isset($html2wp_settings["forms"]) && !empty($html2wp_settings["forms"]) ) {
```

Comment:
Please revise https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage and https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#single-and-double-quotes

Should this check be a function as it's used in multiple places?